inst/www/flex_dashboard/flexdashboard.js

```diff
@@ -1348,13 +1354,14 @@ window.FlexDashboardUtils = {
   },
   manageActiveNavbarMenu: function () {
     // remove active from anyone currently active
-    $('.navbar ul.nav').find('li').removeClass('active');
+    var sel = window.FlexDashboardUtils.isBS3 ? 'li': 'li > a';
+    $('.navbar ul.nav').find(sel).removeClass('active');
```

Comment:
Maybe could just change this to `$('.navbar ul.nav .active').removeClass("active")`